site/installing.md

```diff
@@ -169,13 +169,16 @@ To enable DaemonSets in an existing cluster, add the `--runtime-config=extension
 
 **Install Scope on Your Cluster**
 
-It is recommended that you run Scope natively in your Kubernetes cluster using the manifest generator service.
+It is recommended that you run Scope natively in your Kubernetes cluster using `kubectl` with launch URL as show below.
 
 The simplest way to get the latest release of Scope deployed onto a Kubernetes cluster is the following:
 
-    kubectl create -f 'https://scope.weave.works/launch/k8s/weavescope.json' --validate=false
+    kubectl --namespace=kube-system create -f 'https://scope.weave.works/launch/k8s/weavescope.json' --validate=false
```

Comment:
First of all, `kube-system` is not strictly mandatory, and might not exist, good example is OpenShift. In OpenShift Scope would get deployed into it's own namespace.